tests/test_hra.py

```diff
@@ -973,6 +986,49 @@ def test_layer_no_projection(self):
         actual_message = str(cm.exception)
         self.assertTrue(expected_message in actual_message, actual_message)
 
+    def test_layer_geographic(self):

```

Comment:
Do you mind naming this `test_layer_geographic_projection` so that it's kind of linked by name to the very similar test named `test_layer_no_projection`?